mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -580,4 +401,170 @@ impl Nexus {
             .for_each(drop);
         blockcnt
     }
+
+    /// Starts a rebuild task in the background
+    pub async fn start_rebuild_rpc(&mut self, name: &str) -> Result<(), Error> {
+        if let Err(e) = self.start_rebuild(name).await {
+            Err(e)
+        } else {
+            Ok(())
+        }
+    }
```

Comment:
you can just use an early return here to avoid the else: 
`self.start_rebuild(name).await?;
  Ok(())

Reply:
yeah good catch thanks